src/base/bittorrent/sessionimpl.cpp

```diff
@@ -1982,6 +1983,8 @@ lt::settings_pack SessionImpl::loadLTSettings() const
     settingsPack.set_bool(lt::settings_pack::rate_limit_ip_overhead, includeOverheadInLimits());
     // IP address to announce to trackers
     settingsPack.set_str(lt::settings_pack::announce_ip, announceIP().toStdString());
+    // Port to announce to trackers
+    settingsPack.set_int(lt::settings_pack::announce_port, announcePort());
```

Comment:
You should guard against libtorrent version here and only here:
```suggestion
#if LIBTORRENT_VERSION_NUM >= XXXXX // please fill the correct version in
    // Port to announce to trackers
    settingsPack.set_int(lt::settings_pack::announce_port, announcePort());
#endif
```

Reply:
Thanks a lot for the review. I will resolve this item once the merged https://github.com/arvidn/libtorrent/pull/7771 reaches a release.